shared/settings/render.js.flow

```diff
@@ -7,7 +7,6 @@ export type Props = {
   children: React.Node,
   badgeNumbers: {[key: Tab]: number},
   selectedTab: Tab,
-  showComingSoon: boolean,
```

Comment:
seems unused

Reply:
should remove it here also: https://github.com/keybase/client/blob/264a242ee310ba7d5e6faa1a1722225176f9557d/shared/settings/dumb.desktop.js#L233

Reply:
Done.